### ISSUE TITLE
Multiday events display as "+1 more" with eventLimit of 1

### ISSUE THREAD

**Issue by chris45560** [fullcalendar/fullcalendar#4960]
Hello everyone,

As you can see in the codepen just below : 
https://codepen.io/anon/pen/YmezVR
The different events are displayed correctly.

But, if i had this simple code : 
```
views: {
                dayGridMonth: {
                  eventLimit: 3,
                },
              },
```
I obtain this result : 
![Capture d’écran 2019-08-07 à 09 40 32](https://user-images.githubusercontent.com/25120162/62604056-86d43d80-b8f7-11e9-9aec-76101b705785.png)

As you can see, the 12th of August, i have only 2 events which are displayed, not 3 !

And, if i change it to : 
```
views: {
                dayGridMonth: {
                  eventLimit: 1,
                },
              },
```
I have this display : 
![Capture d’écran 2019-08-07 à 09 42 26](https://user-images.githubusercontent.com/25120162/62604142-aff4ce00-b8f7-11e9-9e0c-f85f1f4550fe.png)

I have '+1 more' everyday ! 
Is there a solution ? Is there a parameter i don't know ?

Thank you for your time !

**Comment by acerix** [comment]
`eventLimit` sets the maximum number of _rows_; each even is one row but "+n events" is also a row.

**Comment by chris45560** [comment]
Ok, i understand this point thank you but, why i have _'+1 more'_ the 7th and the 8th of August instead of displayed the event itself ?

**Comment by acerix** [comment]
It makes sense that if the limit is 1 row, and there is only 1 event, it should show the event in that row instead of "+1 more".  That seems like a bug since other days in your example do show the event in that case.

https://codepen.io/acerix/pen/xvjVXK?editors=0010

There's a related issue to rename eventLimit and add an option that limits the number of events:

https://github.com/fullcalendar/fullcalendar/issues/3035

**Comment by jonathan-vallet** [comment]
Hi! I got the same bug with eventLimit of 3:

![image](https://user-images.githubusercontent.com/5329099/62856422-2d07b500-bcf5-11e9-994c-578386ffe36f.png)

The event "Titre Rappel" is "interrupted" on August, 12th, then the full line of this event is displayed with "+1 more" instead of displaying the event.

When I took a look at the HTML, the <tr> of this line contains a <td class="fc-event-container fc-limited" colspan="7">, which is not displayed at all, and one cell for every "+1"/"+2". I think we should got the "+2" on 12th, then the event without "fc-limited" on other days.
Good luck to fix this bug!
